source/features/bypass-checks.tsx

```diff
@@ -8,11 +8,11 @@ async function init() {
 }
 
 async function bypass(check) {
-	const details = select('.status-actions', check.parentNode);
+	const details = select<HTMLAnchorElement>('.status-actions', check.parentNode);
 	const response = await fetch(details.href);
 	const dom = domify(await response.text());
 	const directLink = select('a.text-small .octicon-link-external', dom);
-	details.href = directLink.parentNode.href;
+	details.href = (directLink.parentNode as HTMLAnchorElement).href;
```

Comment:
Sometimes, it is inevitable that you have to cast. In this case, the reason we need to cast is because we'll get a type error `Property 'href' does not exist on type 'Node & ParentNode'.ts(2339)` and we know that the `parentNode` is an anchor element.

Reply:
Try 

```sh
npm install select-dom@4.2.0-0
```

Reply:
It won’t avoid the type casting but we can use `parentElement` here too